fileio/easyio.go

```diff
@@ -80,6 +80,9 @@ func EasyOpen(filename string) *EasyReader {
 // EasyCreate creates a file with the input name. Panics if errors are encountered.
 func EasyCreate(filename string) *EasyWriter {
 	answer := EasyWriter{}
+	if filename == "" {
```

Comment:
Nice catch. Could you consider moving this to fileio.MustCreate instead of EasyCreate so that other derivatives of MustCreate can benefit as well?

Reply:
Sounds good to me. I made the switch.